src/service/resources-impl.js

```diff
@@ -732,10 +740,12 @@ export class Resources {
         const iniBox = resource.getInitialLayoutBox();
         const diff = request.newHeight - box.height;
         if (diff == 0) {
-          // Nothing to do.
+          // Nothing to resize, resolve attemptChangeSize promise.
+          if (request.callback) {
```

Comment:
@jridgewell  I leave the callback(true) for `diff=0` here to call `continue` as soon as possible.
Also I think `diff=0` should correspond  to promise resolve, though we don't do anything here.